docs/data_sources/wip_future.rst

```diff
@@ -36,16 +32,6 @@ integration, don't hesitate to `open an issue on Github
 <https://github.com/catalyst-cooperative/pudl/issues>`__ requesting the data and
 explaining why it would be useful.
 
-.. _data-censusdp1tract:
-
-Census DP1
-^^^^^^^^^^
-The `US Census Demographic Profile 1 (DP1) <https://www.census.gov/geographies/mapping-files/2010/geo/tiger-data.html>`__
```

Comment:
I think we want this to live somewhere, rather than getting rid of it completely - it's the only place where we explain our source of census data, and census data source is an often annoying technicality that users might actually care about.

Reply:
I was not thrilled about this move (on my part) either, it just didn't seem to fit in anywhere else.

In general I hope we can get to where every data source has its own dedicated documentation page, but the current template / framework makes that kind of janky and hard to update.

Should I make a non-templated page for it do you think?